### PR TITLE
Updating -Tag docs per Issue #4020

### PR DESCRIPTION
**Pull request by sdwheeler**
## Description
The Tag parameter syntax changed in v2.x but the docs were never updated. 

See https://github.com/Azure/azure-powershell/issues/4020
---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md).**
- [ ] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/preview/documentation/help-generation.md).**
- [ ] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/preview/documentation/cleaning-up-commits.md).
- [x] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.
- [ ] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [ ] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#supportsshouldprocess).
- [ ] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [ ] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [ ] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [ ] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

### CODE REVIEW

File: src/ResourceManager/Automation/Commands.Automation/help/Import-AzureRMAutomationRunbook.md [hunk 1]
```diff
@@ -19,16 +19,17 @@ Import-AzureRmAutomationRunbook [-Path] <String> [-Description <String>] [-Name
...
+graphical runbook (.graphrunbook) file for graphical runbooks. The name of the file becomes the
+name of the runbook. For wps_2 Workflow runbooks, the script must contain a single wps_2 Workflow
+definition that matches the name of the file.
 
 ## EXAMPLES
 
 ### Example 1: Import a runbook from a file
 ```
-PS C:\>$Tags = @{"tag01"='value01"; "tag02"="value02"}
+PS C:\> $Tags = @{"tag01"='value01"; "tag02"="value02"}
```
Comment (markcowl) [inline src/ResourceManager/Automation/Commands.Automation/help/Import-AzureRMAutomationRunbook.md]: There is a single quote in here that doesn't belong

File: src/ResourceManager/KeyVault/Commands.KeyVault/help/Set-AzureKeyVaultSecret.md [hunk 2]
```diff
@@ -19,40 +19,44 @@ Set-AzureKeyVaultSecret [-VaultName] <String> [-Name] <String> [-SecretValue] <S
...
 
-The second command modifies value of the secret named ITSecret in the key vault named Contoso.
-The secret value becomes the value stored in $Secret.
+The second command modifies value of the secret named ITSecret in the key vault named Contoso. The
+secret value becomes the value stored in $Secret.
 
 ### Example 2: Modify the value of a secret using custom attributes
 ```
-PS C:\>$Secret = ConvertTo-SecureString -String 'Password' -AsPlainText -Force 
+PS C:\>$Secret = ConvertTo-SecureString -String 'Password' -AsPlainText -Force
```
Comment (markcowl) [inline src/ResourceManager/KeyVault/Commands.KeyVault/help/Set-AzureKeyVaultSecret.md]: This needs a leading space

**Comment by markcowl** [comment]
@sdwheeler This has conflicts, so we likely cannot take the change until after the release.  Please update to remove conflicts

**Comment by markcowl** [comment]
@sdwheeler there are merge conflicts in md files, please merge and resolve conflicts

**Comment by sdwheeler** [comment]
@markcowl @cormacpayne  I don't know how to resolve these conflicts. Can someone help me out here?

**Pull request merged**